packages/terra-overlay/src/terra-dev-site/doc/overlay/Overlay.1.doc.jsx

```diff
@@ -1,56 +0,0 @@
-import React from 'react';
-import DocTemplate from 'terra-doc-template';
-import ReadMe from '../../../../docs/README.md';
```

Comment:
This README should have been removed since the content was moved to https://github.com/cerner/terra-core/pull/2878/files#diff-3cf6797dd9c8d1499959bae533de80d1R12

Reply:
done here https://github.com/cerner/terra-core/pull/2878/commits/17082193960c6ecc5da77a7adfd102efc4a95d1c.